infra/new_relic/.env.sample

```diff
@@ -0,0 +1,3 @@
+NEW_RELIC_ACCOUNT_ID=1234
+NEW_RELIC_API_KEY=NRAK-**
```

Comment:
is this ok? in the `config/newrelic.yml` you are changing `NEW_RELIC_API_KEY` for `NEW_RELIC_LICENSE_KEY`